scripts/ci/podman-test.sh

```diff
@@ -25,7 +25,10 @@ make install
 popd
 rm -rf "${tmp_dir}"
 
-podman info
+# overlaysfs behaves differently on Ubuntu and breaks CRIU
+# https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1857257
```

Comment:
Should we open a new bug? This one has "Fix Released" status.

Reply:
If you want you can do it. As it is reintroduced with every kernel upgrade I have no high hopes that this will be fixed correctly at some point.

Reply:
Sure, let's try one more time.

Reply:
Sounds good